synapse/models/infotech.py

```diff
@@ -632,8 +632,8 @@ def getModelDefs(self):
                     'doc': 'A GUID that represents a host or system.'}),
 
                 ('it:log:event:type:taxonomy', ('taxonomy', {}), {
```

Comment:
see L1195 docstring for this prop ("A taxonometric type for the log event.")
suggest "The type of log event."